scripts/publish_docs.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
+
+### Fetch the gh-pages branch and switch to it ###
+git fetch origin gh-pages
+git checkout -qf FETCH_HEAD
+git checkout -f gh-pages || git checkout -bf gh-pages
+
+### Setting git author ###
+if [[ "`git config user.name`" == "" ]]
+then
+    git config user.email "noreply@pelagicore.com"
+    git config user.name "Jenkins"
+fi
+
+### Moving docs to root directory and committing ###
+
+API_DOCS="api-docs"
+USER_DOCS="user-docs"
+
+git status
+git rm -rf ${API_DOCS} ${USER_DOCS} || true
+mkdir ${API_DOCS}
+mkdir ${USER_DOCS}
+cp -r build/doc/doxygen-docs/html/* ${API_DOCS}
+cp -r build/doc/html/* ${USER_DOCS}
```

Comment:
This can be simplified into `cp -r build/doc/doxygen-docs/html ${API_DOCS}` and then you don't need the mkdir